Gems/MiniAudio/Code/Source/Clients/MiniAudioPlaybackComponentConfig.h

```diff
@@ -48,7 +49,20 @@ namespace MiniAudio
 
         bool m_enableSpatialization = false;
         AttenuationModel m_attenuationModel = AttenuationModel::Inverse;
-        float m_minimumDistance = 3.f;        
+        float m_minimumDistance = 3.f;
         float m_maximumDistance = 30.f;
+
+        //! Inner cone angle
+        float m_innerAngleInRadians = AZ::Constants::TwoPi;
+        float m_innerAngleInDegrees = m_innerAngleInRadians * 360.f/AZ::Constants::TwoPi;
```

Comment:
Same comment here.  Just a Nit though, wouldn't hold this back just to change it.  The math is still correct.